mod.sp

```diff
@@ -7,6 +7,69 @@ locals {
   }
 }
 
+variable "common_dimensions" {
+  type        = list(string)
+  description = "A list of common dimensions to add to each control."
+  # Define which common dimensions should be added to each control.
+  # - connection_name (_ctx ->> 'connection_name')
+  # - region
+  # - resource_group
+  # - subscription
+  # - subscription_id
+  default = ["resource_group", "region", "subscription"]
```

Comment:
Is region available as a common column for all the azure tables? I don't see region being used in the queries at all.

Reply:
Yes we should include `region` as default, region is available in most of the azure tables and wherever we do not have region column in the table or the resource is global we use `common_dimensions_global_qualifier_sql`. If we remove region from default and spvars it would not show us the region data for those queries which have region column.